test/e2e_node/garbage_collector_test.go

```diff
@@ -231,7 +231,7 @@ func containerGCTest(f *framework.Framework, test testRun) {
 		AfterEach(func() {
 			for _, pod := range test.testPods {
 				By(fmt.Sprintf("Deleting Pod %v", pod.podName))
-				f.PodClient().DeleteSync(pod.podName, &metav1.DeleteOptions{}, defaultRuntimeRequestTimeoutDuration)
+				f.PodClient().DeleteSync(pod.podName, &metav1.DeleteOptions{}, podDisappearTimeout)
```

Comment:
Yeah. I feel we can add default timeouts to DeleteSync() itself which can derive that from the framework. Less constants to deal with in such a workflow.

if this PR is urgent, don't block on these comments.